test/necb/unit_tests/expected_results/pv_ground_expected_results.json

```diff
@@ -5,7 +5,7 @@
     "building_type": "FullServiceRestaurant",
     "primary_heating_fuel": "DefaultFuel",
     "pv_ground_type": true,
-    "Generator PVWatts 1 - dc_system_capacity": "8611.12833336778",
```

Comment:
wow big change.. was this a correction from mike?

Reply:
No, previously I set the default area of the PV panels as 50 m2, whereas I set it to a building's footprint in my revisions.